master/usage/ip-pool-manipulation.md

```diff
@@ -0,0 +1,82 @@
+---
+title: IP Pool Manipulation
+---
+The 
+[IP Pools]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool)
+that Calico starts up with can be manipulated and changed.  This document
+describes how to do that using the `calicoctl` utility.
+
+## Retrieving IP Pool information
+
+The current IP Pools can be retrieved with the following commmand.
+
+```shell
+calicoctl get ipPool
+```
+
+This will return information similar to the following.
```

Comment:
Combine this with the section above by showing command and output in the same section, like so:
```
$ calicoctl get ippool
CIDR                       
192.0.2.0/24               
fd80:24e2:f998:72d6::/64   
```